pallets/utxo/rpc/runtime-api/src/lib.rs

```diff
@@ -25,6 +26,8 @@ sp_api::decl_runtime_apis! {
         // At the moment we have some problems with use serde in RPC, we can serialize and deserialize
         // only simple types. This approach allow us to return Vec<(TokenId, TokenName)> instead of
         // pallet_utxo_tokens::TokenListData
-        fn tokens_list() -> Vec<(u64, Vec<u8>)>;
+        fn tokens_list() -> Vec<(H256, Vec<u8>)>;
```

Comment:
I think this function will only be useful for debugging. In reality, no application in the world will require having the list of ALL tokens on a blockchain. Imagine having that on Ethereum...

Reply:
I think the chance of this surviving in to the testnet once we're happy everything is working properly is very slim

Reply:
Removed